.env

```diff
@@ -0,0 +1 @@
+GRPC_WEB_API_URL=http://localhost:8080
```

Comment:
i suggest to not upload the .env file, if u want to upload, please create a copy file named `env.example` that doesn't contain real data. example:

```
GRPC_WEB_API_URL=
```